framer/Events.coffee

```diff
@@ -25,15 +25,14 @@ Events.enableEmulatedTouchEvents = (enable=true) ->
 		Events.TouchStart = "touchstart"
 		Events.TouchEnd = "touchend"
 		Events.TouchMove = "touchmove"
+	Events.Click = Events.TouchEnd
 
 Events.enableEmulatedTouchEvents(false)
 
-# Let's make sure the touch events work on desktop too
-if not Utils.isTouch()
+# Let's make sure the touch events work with a mouse too
+if Utils.isMouse()
```

Comment:
I now see there’s an extra guard in the enable. Maybe this `if` should be removed and make the logic for enabling live purely there?